_posts/2020-10-07-presto-first-steps.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: post
+title:  Launching Presto First Steps training
+author: Manfred Moser
+excerpt_separator: <!--more-->
+---
+
+Writing the book [Presto: The Definitive
+Guide](/presto-the-definitive-guide.html) with Matt and Martin earlier this
+year, and then publishing it with [O'Reilly](https://www.oreilly.com/) was a
+great experience and has been a great success. Lots of readers took advantage of
+getting a [free digital copy of the book from Starburst({% post_url
+2020-04-11-the-definitive-guide %}).
+
+Now it is time to follow up with a training class. I am pleased to let you know
+that you can join me for three hours of
+[Presto First Steps](https://learning.oreilly.com/live-training/courses/presto-first-steps/0636920462859/)
+in November.
+
+<!--more-->
+
+The new course is aimed at beginners with Presto, who want to accelerate their
+initial understanding and adoption. You ramp up quickly to install and configure
+Presto, use the CLI, and learn how to query connected data sources with SQL.
+
+The class includes three interactive training exercises on Katacoda. They allow
```

Comment:
A link to katacoda.com here might be useful for those who don't know about it.

Reply:
done